.github/workflows/continuous-integration-workflow.yml

```diff
@@ -75,11 +77,11 @@ jobs:
                 -DKokkos_ENABLE_OPENMP=ON && \
               cmake --build builddir --parallel 2 && \
               ctest --test-dir builddir --output-on-failure"
+          sed -i '26s/$/ /' ${{ matrix.config.dockerfile }} # force rebuild by adding a space (kokkos/ci-containers#56) 
```

Comment:
Why line 26?
https://github.com/kokkos/ci-containers/blob/79beb832f82545f95397911e9f380d92c9396c80/ubuntu#L26
https://github.com/kokkos/ci-containers/blob/79beb832f82545f95397911e9f380d92c9396c80/fedora#L26
The opensuse file only has 13 lines.

Does that mean prepend a space to the 26th line?
If so please update the comment so that I don't wonder again what this is actually doing next time I look.

Reply:
It was because it fails on line 26 for ubuntu.